### ISSUE TITLE
trial.test.test_runner.ExcInfoHoldingErrorHolderTests.test_repr: extra , in repr()

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#9524]
|[<img alt="altendky's avatar" src="https://avatars.githubusercontent.com/u/543719?s=50" width="50" height="50">](https://github.com/altendky)| @altendky reported|
|-|-|
|Trac ID|trac#9524|
|Type|defect|
|Created|2018-09-06 13:30:24Z|

https://travis-ci.org/twisted/twisted/jobs/425258725#L3376

https://ci.appveyor.com/project/adiroiban/twisted/build/trunk-5347/job/yuqn8cgxx794pst3#L18061

```python
[FAIL]
Traceback (most recent call last):
  File "c:\projects\twisted\build\py37-alldeps-withcov-windows\lib\site-packages\twisted\trial\test\test_runner.py", line 848, in test_repr
    "<ErrorHolder description='description' "
  File "c:\projects\twisted\build\py37-alldeps-withcov-windows\lib\site-packages\twisted\trial\_synctest.py", line 432, in assertEqual
    super(_Assertions, self).assertEqual(first, second, msg)
  File "C:\PYTHON37-x64\Lib\unittest\case.py", line 839, in assertEqual
    assertion_func(first, second, msg=msg)
  File "C:\PYTHON37-x64\Lib\unittest\case.py", line 1220, in assertMultiLineEqual
    self.fail(self._formatMessage(msg, standardMsg))
twisted.trial.unittest.FailTest: "<Err[36 chars]rror=ZeroDivisionError('integer division or modulo by zero')>" != "<Err[36 chars]rror=ZeroDivisionError('integer division or modulo by zero',)>"
- <ErrorHolder description='description' error=ZeroDivisionError('integer division or modulo by zero')>
+ <ErrorHolder description='description' error=ZeroDivisionError('integer division or modulo by zero',)>
?                                                                                                    +
twisted.trial.test.test_runner.ExcInfoHoldingErrorHolderTests.test_repr
```

<details><summary>Searchable metadata</summary>

```
trac-id__9524 9524
type__defect defect
reporter__altendky altendky
priority__normal normal
milestone__Python_3_7_support Python 3.7 support
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__trial trial
keywords__None None
time__1536240624314546 1536240624314546
changetime__1536333086177786 1536333086177786
version__None None
owner__rodrigc rodrigc

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="altendky's avatar" src="https://avatars.githubusercontent.com/u/543719?s=50" width="50" height="50">](https://github.com/altendky)|@altendky set owner to @altendky|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="altendky's avatar" src="https://avatars.githubusercontent.com/u/543719?s=50" width="50" height="50">](https://github.com/altendky)<a name="note_2"></a>|@altendky commented|
|-|-|

The builtin `ZeroDivisionError` changed it's own representation.  

https://github.com/twisted/twisted/pull/1058

```python
Python 3.6.6 (default, Jul 24 2018, 16:23:12) 
[GCC 6.3.0 20170516] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> repr(ZeroDivisionError('abc'))
"ZeroDivisionError('abc',)"
>>>
```

```python3
Python 3.7.0 (default, Jul  7 2018, 15:49:24) 
[GCC 6.3.0 20170516] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> repr(ZeroDivisionError('abc'))
"ZeroDivisionError('abc')"
>>>
```

**Comment by twisted-trac** [comment]
|[<img alt="altendky's avatar" src="https://avatars.githubusercontent.com/u/543719?s=50" width="50" height="50">](https://github.com/altendky)<a name="note_3"></a>|@altendky set owner to @rodrigc|
|-|-|

How do you like rev2?

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_4"></a>|@rodrigc commented|
|-|-|

Approved: https://github.com/twisted/twisted/pull/1058#pullrequestreview-153194547

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_5"></a>|@rodrigc set status to `closed`|
|-|-|

In changeset 121c5312da1037700ef3adc18af4c3bfc4fd8eda
```
#!CommitTicketReference repository="" revision="121c5312da1037700ef3adc18af4c3bfc4fd8eda"
Merge pull request #1058 from altendky/9524-altendky-ExcInfoHoldingErrorHolderTests_repr

Author: 	altendky
Reviewer: rodrigc
Fixes: ticket:9524

Fix test on Python 3.7
```
